integration-tests/gradle/src/main/resources/multi-source-project/src/main/java/org/acme/service/SimpleService.java

```diff
@@ -6,7 +6,6 @@
 
 @ApplicationScoped
 public class SimpleService {
-    @NotNull
```

Comment:
Don't forget to remove the import too

Reply:
I thought Maven already did, but I guess it didn't for these gradle tests